docs/docs/recipes.md

```diff
@@ -2020,113 +2020,113 @@ export const query = graphql`
+5. Certifique-se de que o seu site tenha a mesma aparência ao executar o `gatsby build` como no `gatsby develop`. Executando `gatsby serve` ao criar seu site, você pode testar (e depurar, se necessário) o produto final antes de implantá-lo ao vivo.
 
 ```shell
 gatsby build && gatsby serve
 ```
 
-#### Additional resources
+#### Recursos adicionais
 
-- Walk through building and deploying an example site in [tutorial part one](/tutorial/part-one/#deploying-a-gatsby-site)
-- Learn about [performance optimization](/docs/performance/)
-- Read about [other deployment related topics](/docs/preparing-for-deployment/)
-- Check out the [deployment docs](/docs/deploying-and-hosting/) for specific hosting platforms and how to deploy to them
+- Caminho para criação e implantação de um site de exemplo no [tutorial parte um](/tutorial/part-one/#deploying-a-gatsby-site)
+- Aprenda sobre [otimização de desempenho](/docs/performance/)
+- Leia sobre [outros tópicos relacionados à implantação](/docs/preparing-for-deployment/)
+- Confira os [documentos de implantação](/docs/deploying-and-hosting/) para plataformas de hospedagem específicas e como implantar nelas
 
-### Deploying to Netlify
+### Implantando em Netlify
 
-Use [`netlify-cli`](https://www.netlify.com/docs/cli/) to deploy your Gatsby application without leaving the command-line interface.
+Use [`netlify-cli`](https://www.netlify.com/docs/cli/) para implantar seu aplicativo Gatsby sem sair da interface da linha de comandos.
 
-#### Prerequisites
+#### Pré-requisitos
 
-- A [Gatsby site](/docs/quick-start) with a single component `index.js`
-- The [netlify-cli](https://www.npmjs.com/package/netlify-cli) package installed
-- The [Gatsby CLI](/docs/gatsby-cli) installed
+- Um [site Gatsby](/docs/quick-start) com um único componente `index.js`
+- O pacote [netlify-cli](https://www.npmjs.com/package/netlify-cli) instalado
+- A [ILC Gatsby](/docs/gatsby-cli) instalada
 
-#### Directions
+#### Instruções
 
-1. Build your gatsby application using `gatsby build`
+1. Crie seu aplicativo gatsby usando `gatsby build`
 
-2. Login into Netlify using `netlify login`
+2. Entre no Netlify usando `netlify login`
 
-3. Run the command `netlify init`. Select the "Create & configure a new site" option.
+3. Execute o comando `netlify init`. Selecione a opção "Create & configure a new site".
 
-4. Choose a custom website name if you want or press enter to receive a random one.
+4. Escolha um nome de site personalizado, se desejar, ou pressione enter para receber um nome aleatório.
 
-5. Choose your [Team](https://www.netlify.com/docs/teams/).
+5. Escolha seu [Team](https://www.netlify.com/docs/teams/).
 
-6. Change the deploy path to `public/`
+6. Mude o caminho de implantação para `public/`
 
-7. Make sure that everything looks fine before deploying to production using `netlify deploy --prod`
+7. Verifique se tudo está bem antes de implantar na produção usando `netlify deploy --prod`
```

Comment:
melhorar a traducao de 'deploying to production'.
Sugestão:  "disponibilizar em/no ambiente de produção"